bin/resources/local.rb

```diff
@@ -175,7 +182,6 @@ def start_local_server(project_directory, mongo_directory, ruby_path, worker_num
 
   state_file = ::File.join(project_directory, cluster_name + '_configuration.json')
   receipt_file = ::File.join(project_directory, cluster_name + '_configuration.receipt')
```

Comment:
Also do we need a separate server.pid?  Seems like this is in the configuration.json file